<launch>
  <node name="simMaccepa" pkg="maccepavd" type="simulator.py"/>
  
  <node name="call_sensors_server" pkg="maccepavd" type="sensor_server.py"/>
  <node name="exectj_forward_action_server" pkg="maccepavd" type="exectj_forward_aserver.py" />
  
</launch>
